hafnian/quantum.py

```diff
@@ -609,40 +609,38 @@ def gen_Qmat_from_graph(A, n_mean):
     return Q
 
 
-def is_valid_cov(cov, hbar=2, sigdigits=6):
+def is_valid_cov(cov, hbar=2, rtol=1e-05, atol=1e-08):
     r""" Checks if the covariance matrix is a valid quantum covariance matrix.
 
     Args:
         cov (array): a covariance matrix
         hbar (float): value of hbar in the uncertainty relation
 
     Returns:
-        (boolean): whether the given covariance matrix is a valid covariance matrix
+        (bool): whether the given covariance matrix is a valid covariance matrix
     """
     (n, m) = cov.shape
     if n != m:
         # raise ValueError("The input matrix must be square")
         return False
-
-    if np.linalg.norm(cov - np.transpose(cov)) >= 10 ** (-sigdigits):
+    if not np.allclose(cov, np.transpose(cov), rtol=rtol, atol=atol):

```

Comment:
new lines seem to be disappearing 😕 
```suggestion

    if not np.allclose(cov, np.transpose(cov), rtol=rtol, atol=atol):
```